processor/src/main/java/org/mapstruct/ap/MappingProcessor.java

```diff
@@ -63,7 +62,7 @@
  * <ul>
  * <li>retrieve mapping methods</li>
  * <li>create the {@code Mapper} model</li>
- * <li>perform enrichments and modifications (e.g. add annotations for dependency injection)</li>
+ * <li>perform enrichment's and modifications (e.g. add annotations for dependency injection)</li>
```

Comment:
Why do you think that we need to change this? Right now it is a plural noun of enrichment. With the change it becomes something else.

Reply:
I agree @filiphr  .Looks like the apostrophe came in from intellij and sonar plugin just before pushing the change.